README.md

```diff
@@ -93,12 +93,16 @@ require('alexa-app-server').start({
 	// root of your web server.
 	app_root: "/alexa/",
 
-	// The directory containing server-side processing modules (see below)
+	// The directory containing server-side processing modules (see below).
 	server_dir: "server",
 
-	// The port the server should bind to
+	// The port the server should bind to.
 	port: 80,
 
+	// The host address in which the server should bind to. If not specified,
+	// this argument will be ignored.
+	host_address: undefined,
```

Comment:
Should we just default this to 127.0.0.1 and avoid binding without a host name and therefore preventing remote access by default?